tide_landing_page.info.yml

```diff
@@ -296,6 +296,7 @@ config_devel:
     - paragraphs.paragraphs_type.latest_events
     - paragraphs.paragraphs_type.media_gallery
     - paragraphs.paragraphs_type.contact_us
+    - paragraphs.paragraphs_type.news_listing
```

Comment:
@vincent-gao I think you're right, it was weird that was down in optional. Have moved up here.